lint/src/main/kotlin/net/grandcentrix/thirtyinch/Issues.kt

```diff
@@ -0,0 +1,31 @@
+package net.grandcentrix.thirtyinch
+
+import com.android.tools.lint.detector.api.*
+import java.util.*
+
+enum class Issues(
```

Comment:
The `Issues` type is supposed to declare the limited set of available detections that the ThirtyInch Lint module can analyze. Both `data` and "normal" classes wouldn't easily allow for this defined set of options, unless we used constants, which I wouldn't encourage. A `sealed class` could work, yet each implementation would have the same duplicated properties anyway.